app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -402,6 +404,38 @@ public void setCurrentSession(int aId) {
             mCurrentSession.setActive(true);
     }
 
+    public String setRegion(String aRegion) {
+        Log.d(LOGTAG, "SessionStore setRegion: " + aRegion);
+        String result = "worldwide";
```

Comment:
perhaps I should move the log statement to reflect what the region could be

Reply:
Not sure why does setRegion need to return the region that's being set.

Reply:
thanks, good call - yeah, it doesn't need to